recipes/multimodal/server/backends/salm_backend.py

```
@@ -0,0 +1,200 @@
+                wavf.writeframes(b"\x00\x00" * 16000)
+            prompts = [
+                [
+                    {
+                        "role": "user",
+                        "content": f"{self.salm_config.user_prompt} {self._model.audio_locator_tag}",
+                        "audio": [path],
+                    }
+                ]
+            ]
+            self._model.generate(prompts=prompts, max_new_tokens=8)
+        except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)